### PR TITLE
[DSD-5387] added certify properties for different plugins

### PR DESCRIPTION
**Pull request by challabeehyv**

### CODE REVIEW

File: certify-sunbird-insurance.properties [hunk 1]
```diff
@@ -1,41 +1,43 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
```
Comment (vishwa-vyom) [inline certify-sunbird-insurance.properties:3]: Lets have the license text in all the config files

File: certify-sunbird-insurance.properties [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
+mosip.certify.integration.vci-plugin=SunbirdRCVCIssuancePlugin
+mosip.certify.integration.audit-plugin=LoggerAuditService
 
+## ------------------------------------------- Plugin specific usecase properties ------------------------------------------------------------
+mosip.certify.vciplugin.sunbird-rc.enable-psut-based-registry-search=false
+mosip.certify.vciplugin.sunbird-rc.issue-credential-url=http://10.3.148.107/credential/credentials/issue
+mosip.certify.vciplugin.sunbird-rc.supported-credential-types=LifeInsuranceCredential,InsuranceCredential,VehicleInsuranceCredential
 
-## -------------------------------------- Authentication & Authorization -----------------------------------------------
+  ##---------------------------------Life Insurance Credential Configurations------------------------------------------------------
```
Comment (vishwa-vyom) [inline certify-sunbird-insurance.properties:11]: Lets have this as comment without --- line so the sections are not confused

File: certify-sunbird-insurance.properties [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
+  ##---------------------------------Vehicle Insurance Credential Configurations------------------------------------------------------
 
-mosip.certify.security.ignore-auth-urls=/actuator/**,**/error,**/swagger-ui/**,\
-  **/v3/api-docs/**, **/issuance/**
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/vehicle-insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.cred-schema-id=did:schema:e95a24f5-d65e-4b65-b497-e0671f8bd12d
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.VehicleInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```
Comment (vishwa-vyom) [inline certify-sunbird-insurance.properties:27]: Do we need VehicleInsuranceCredential ? Is Life and Health Insurance are not our defaults ?

File: certify-sunbird-insurance.properties [hunk 4]
```diff
@@ -1,41 +1,43 @@
...
 
-mosip.certify.security.auth.post-urls={}
-mosip.certify.security.auth.put-urls={}
-mosip.certify.security.auth.get-urls={}
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.static-value-map.issuerId=did:web:holashchand.github.io:test_project:32b08ca7-9979-4f42-aacc-1d73f3ac5322
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.template-url=${spring_config_url_env}/*/${active_profile_env}/${spring_config_label_env}/life-insurance-credential.json
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-get-url=http://10.3.148.107/registry/api/v1/Insurance/
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-id=did:schema:9ae73412-5783-4194-acec-6d004006660c
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.cred-schema-version=1.0.0
+mosip.certify.vciplugin.sunbird-rc.credential-type.LifeInsuranceCredential.registry-search-url=http://10.3.148.107/registry/api/v1/Insurance/search
```
Comment (vishwa-vyom) [inline certify-sunbird-insurance.properties:18]: Can we have this and other IPs changed to internal service urls ?

File: certify-sunbird-insurance.properties [hunk 5]
```diff
@@ -102,10 +104,10 @@ mosip.certify.key-values={\
    'vd12' : {\
               'credential_issuer': '${mosip.certify.identifier}',   \
               'authorization_servers': {'${mosip.certify.authorization.url}'}, \
-              'credential_endpoint': '${mosipbox.public.url}${server.servlet.path}/issuance/vd12/credential', \
+              'credential_endpoint': '${mosip.certify.domain.url}${server.servlet.path}/issuance/vd12/credential', \
```
Comment (vishwa-vyom) [inline certify-sunbird-insurance.properties:107]: I think this and identifier should also be overrided here to allow the different URLs

File: certify-mock.properties [hunk 6]
```diff
@@ -1,226 +1,121 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+## ------------------------------------------- Plugin enable properties ------------------------------------------------------------
```
Comment (vishwa-vyom) [inline certify-mock.properties:1]: Are we not removing this file ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: certify-sunbird-insurance.properties (hunk 1 above)
Comment (challabeehyv) [reply certify-sunbird-insurance.properties:3]: added to all files

File: certify-sunbird-insurance.properties (hunk 2 above)
Comment (challabeehyv) [reply certify-sunbird-insurance.properties]: added as a comment

File: certify-sunbird-insurance.properties (hunk 3 above)
Comment (challabeehyv) [reply certify-sunbird-insurance.properties]: removed VehicleInsuranceCredential

File: certify-sunbird-insurance.properties (hunk 5 above)
Comment (challabeehyv) [reply certify-sunbird-insurance.properties:103]: added identifier and domain.url as overrided properties in all plugin property files

File: certify-mock.properties (hunk 6 above)
Comment (challabeehyv) [reply certify-mock.properties:5]: this is the file for mock plugin use case properties

File: certify-sunbird-insurance.properties (hunk 4 above)
Comment (challabeehyv) [reply certify-sunbird-insurance.properties:22]: can we change this after the release, because we haven't tested with internal urls

Comment (vishwa-vyom) [reply certify-sunbird-insurance.properties]: ok

File: certify-mosipid-identity.properties [hunk 7]
```diff
@@ -25,6 +29,9 @@ mosip.certify.cache.security.algorithm-name=AES/ECB/PKCS5Padding
 
 ## ------------------------------------------- Use specific default overriding properties ------------------------------------------------------------
 
+mosip.certify.domain.url=https://${mosip.injicertify.host}
```
Comment (vishwa-vyom) [inline certify-mosipid-identity.properties:32]: I think the value should be different here like https://${mosip.injicertify.mosipid.host}

File: certify-mock.properties (hunk 6 above)
Comment (vishwa-vyom) [reply certify-mock.properties:5]: then it has to be renamed to certify-mock-identity.properties

File: certify-mosipid-identity.properties (hunk 7 above)
Comment (challabeehyv) [reply certify-mosipid-identity.properties]: changed specific to three namespaces

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**